### PR TITLE
[RFR] Fix zoom center

### PR DESCRIPTION
**Pull request by jpetitcolas**
peek 2017-11-13 18-06

### CODE REVIEW

**Comment by jpetitcolas** [comment]
Anyone to review this one? :)

File: src/axis.spec.js [hunk 1]
```diff
@@ -40,21 +40,22 @@ describe('Axis', () => {
...
+        axis(d3, defaultConfig, defaultScale)(selection);
 
         selection.data([[]]);
-        axis(defaultConfig, defaultScale)(selection);
+        axis(d3, defaultConfig, defaultScale)(selection);
         expect(document.querySelectorAll('.axis').length).toBe(0);
     });
 
     afterEach(() => {
+        document.body.innerHTML = '';
```
Comment (Kmaschta) [inline src/axis.spec.js]: :1st_place_medal:

File: src/zoom.js [hunk 2]
```diff
@@ -1,10 +1,41 @@
...
+    const {
+        label: {
+            width: labelsWidth,
+            padding: labelsPadding,
+        },
+    } = config;
+
     const zoom = d3.zoom();
 
+    svg.attr('transform-origin', 'translate(-200)');
```
Comment (Kmaschta) [inline src/zoom.js]: Is this really usefull?

**Comment by jpetitcolas** [comment]
Code review applied.

**Pull request merged**